### DISCUSSION TITLE
NEW Release

### DISCUSSION THREAD

**Discussion by Mohamed-Ziyad** [ViroCommunity/viro#241] [General]
Hi,You are doing a great job for the community, Thank you very much.  When will you release new version for react-native new version?, because I tried to use this ViroReact with react native 0.70+ I can't configure it and most of the articles and blogs say It's not compatible with react native latest version. any solutions?

**Comment by cjmcassar** [comment]
Hi, thanks for letting up know. We've recently set up a core team that will work on this full time. Hopefully we can create a new release in the next month.

**Comment by Mohamed-Ziyad** [comment]
Thank you for the update.
